WORKSPACE

```diff
@@ -130,3 +131,13 @@ java_import_external(
 load("//private:format.bzl", "format_repositories")
 
 format_repositories()
+
+http_archive(
+    name = "bazel_toolchains",
```

Comment:
where is this being used? I don't see it?

Reply:
`rbe_ubuntu1604` is the identifier of a runtime which runs Bazel on a build server while using RBE (remote build execution) for remote execution. Part of the flags which they pass to Bazel includes `bazel_toolchains` for example `extra_toolchains` and `crosstool_top` point there